README.md

```diff
@@ -376,7 +376,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
     * [gomemcache](https://github.com/bradfitz/gomemcache/) - memcache client library for the Go programming language.
     * [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB.
     * [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV.
-    * [mgo](https://godoc.org/labix.org/v2/mgo) - MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
+    * [mgo] (https://godoc.org/github.com/globalsign/mgo) MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms
```

Comment:
remove space between `[mgo]` and `(https://xxxx)`

Reply:
@appleboy oh

completed